api/nodes/views.py

```diff
@@ -521,16 +534,6 @@ class NodeContributorDetail(BaseContributorDetail, generics.RetrieveUpdateDestro
     def get_resource(self):
         return self.get_node()
 
-    # overrides DestroyAPIView
-    def perform_destroy(self, instance):
```

Comment:
As I said above don't bother using a mixin, it's okay to repeat yourself a little across views. It's just easier to see new behavior when you have a small number of methods for a view.